docs/technical_reference/unit_models/zero_order_unit_models/gas_sparged_membrane_zo.rst

```diff
@@ -12,8 +13,25 @@ See documentation for :ref:`Helper Methods for Electricity Demand<electricity_me
 
 Costing Method
 --------------
-Costing is calculated using the False method in the zero-order costing package.
-See documentation for the :ref:`zero-order costing package<zero_order_costing>`.
+This unit does not include costing.
+
+Additional Variables
+--------------------
+
+.. csv-table::
+   :header: "Description", "Variable Name", "Units"
+
+   "Mass flow of gas extracted per mass flow of influent", "gas_mass_influent_ratio", "None"
```

Comment:
Minor comment - would this be better rendered as `dimensionless` than `None`? Whilst they probably mean the same thing to most users, from a technical standpoint they are slightly different (explicitly dimensionless versus no units assigned).

Reply:
FYI the line I am referring to is here:
https://github.com/watertap-org/watertap/blob/b901e76ffdd2badeeabb7055439d3418d2f01337/docs/technical_reference/unit_models/zero_order_unit_models/automate_rst_file_wt3.py#L140

Reply:
No -- dimensionless variables have units of `None`, even if explicitly specified.

We could just replace `"None"` with `"dimensionless"` in our string processing, however, e.g.:
```python
added_var_units.append(str(model_var._units).replace("'", ""). replace("None", "dimensionless")) 
```
or better:
```python
var_units = model_var._units
if var_units is None:
    added_var_units.append("dimensionless")
else:
    added_var_units.append(str(var_units).replace("'", ""))
```

Reply:
Oh, I just saw your comments on this now. I already added something to handle this, though my solution may be more crude than either of your proposed solutions:

https://github.com/watertap-org/watertap/blob/89d6bf629915742588a472e05c7e1100f2bbfe3a/docs/technical_reference/unit_models/zero_order_unit_models/automate_rst_file_wt3.py#L140-L142